src/staging/src/styles/home.scss

```diff
@@ -467,3 +373,89 @@ $tabletBreakpoint: 768px;
 
   padding-bottom: 40px;
 }
+
+/* ------------- SEARCH ------------ */@at-root
```

Comment:
I'm not clear why `@at-root` is necessary here?

Reply:
I think this was a intellisense/typo. Removing.